server/app.js

```diff
@@ -354,6 +356,10 @@ module.exports = function createApp({
     '/admin/licences/',
     secureRoute(licenceRouter(licenceService, signInService, prisonerService, audit, roNotificationHandler))
   )
+  app.use(
+    '/admin/functionalMailboxes',
+    secureRoute(functionalMailboxRouter(functionalMailboxService), { auditKey: 'FUNCTIONAL_MAILBOX' })
```

Comment:
do you need to add this as an allowed audit key: in audit.js?

Reply:
Yes.  You're right. I forgot about auditing. Partly because I couldn't decide where it should be done. (probation-teams or licences).  licences has audit support in place and knows the requesting username, so it would be simple to add it here - which I will do now.